README.md

```diff
@@ -210,12 +210,12 @@ Cruler expects validators to return Map. The Map should be as follow.
 
 `:errors` is array of Map. The Map should be as follows.
 
-| key            | type     | description                                                                                                                                              |
-| -------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `:file-path`   | string   | Relative path from the project's root dir                                                                                                                |
-| `:error-value` | string   | It should be invalid value                                                                                                                               |
```

Comment:
Nice catch! 

However, `:error-value` is handled in https://github.com/xcoo/cruler/blob/main/src/cruler/core.clj#L86-L93.
And `string` is not the very correct type as you found but it could be not only `hash-map` but also `string`.  

For example
+ the case `error-value is string`: https://github.com/xcoo/cruler/blob/main/dev-resources/sample-validator/validator/sample_validator/reference.clj#L22
+ the case `error-value is hash-map`: https://github.com/xcoo/cruler/blob/main/dev-resources/sample-validator/validator/sample_validator/spec.clj#L13

Reply:
I see. I will change the document for we can pass the string and the hash-map to the error-value.